### PR TITLE
made a new text file

### PR DESCRIPTION
**Pull request by gkd2000**

### CODE REVIEW

File: testPage.html [hunk 1]
```diff
@@ -26,4 +26,3 @@
 		</footer>
 	</body>
 </html>
-4
```
Comment (clebio) [inline testPage.html]: 👍

File: grace.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+I'm a mess guys
```
Comment (clebio) [inline grace.txt:1]: Do ya really want that out on FRC1775.github.io/grace.txt? At any rate, a filename ending in `.md` will get styled with [Markdown](https://daringfireball.net/projects/markdown/syntax).

Can `git mv` the file to a new filename and edit, add, commit, push (to this same branch) and this PR will automatically get updated.

Comment (gkd2000) [reply grace.txt:1]: No, I don't want that there. I'm sorry. I didn't submit the pull request intentionally, and I couldn't figure out how to undo it.

Comment (clebio) [reply grace.txt:1]: 👍

**Pull request closed without merging**